src/main/java/com/sprint/mission/discodeit/dto/ChannelDto/ChannelCreatePrivateDto.java

```diff
@@ -0,0 +1,31 @@
+package com.sprint.mission.discodeit.dto.ChannelDto;
```

Comment:
dto 별도 패키지로 분리하신건 좋아보여요 👍   
대신 패키지명은 보통 소문자로만 짓기 때문에 아래처럼 바꿔봐도 좋을 것 같아요.  
```suggestion
package com.sprint.mission.discodeit.dto.channel;
```